Dockerfiles/agent/nosys-seccomp/nosys.c

```diff
@@ -1,3 +1,4 @@
+#define _GNU_SOURCE
```

Comment:
Should add a comment to understand which feature/function call we use that requires `_GNU_SOURCE`

Reply:
Comment added: https://github.com/DataDog/datadog-agent/pull/13019/files#diff-dc1f53b7bf2f627386266884978aa36463a4010ad3772c78ff40372ba272e41bR1.